src/Mod/Path/PathScripts/PathGui.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, widget, obj, prop, onBeforeChange=None):
     def eventFilter(self, obj, event):
         if event.type() == QtCore.QEvent.Type.FocusIn:
             self.updateSpinBox()
+            self.widget.editingFinished.emit()
```

Comment:
`FocusIn` is the event when the widget _gets_ the focus which is not the time to emit `editingFinished`

Reply:
The widget is the SpinBox, not the QLineEdit for the expression, so yes, we need it after the expression editing session is finished with the QLineEdit and focus is returned back to the SpinBox.   If I move the signal emission to `Type == FocusOut`, then we are back where we started with the user having to manually click on the task panel or view port for the code to apply the change to Default Length.